src/pipeline_v3/tests/integration/test_llama_index_availability.py

```
@@ -0,0 +1,74 @@
+        errors.append(f"OpenAIEmbedding: {e}")
+
+    try:
+        from llama_index.vector_stores.qdrant import QdrantVectorStore
+        print("✓ QdrantVectorStore imported successfully")
+    except ImportError as e:
+        errors.append(f"QdrantVectorStore: {e}")
+
+    # Check if we can access the flag from index_manager
+    try:
+        # Try with relative import
+        import sys
```

Pylint (3.3.7)

[convention] Import outside toplevel (sys)

(C0415)